monero-outreach-docs/translations/es/archive/triptych_es.md

```diff
@@ -0,0 +1,43 @@
+
+Los principales algoritmos de las firmas de anillo también han mejorado. El libro blanco de CryptoNote entregó un esquema de firmas de anillos diferente de la actual implementación, basada en firmas con mayor eficiencia de un Grupo Anónimo Multicapa de Vinculación Espontánea (MLSAG por sus siglas en inglés). Una actualización esperada llamada firmas de Grupo Anónimo Conciso de Vinculación Espontánea (CLSAG por sus siglas en inglés) viene en camino y se espera que haga una reducción adicional al tamaño de las transacciones de un cuarto del tamaño actual, y una reducción de verificación en los tiempos por casi una décima parte (para una transacción de dos entradas, dos salidas).
+
+Todas estás actualizaciones le han dan a Monero un anonimato único. Cada TXO real, desconectada criptográficamente de la dirección del remitente, que se gasta en la forma de una llave pública TXO, se oculta entre 10 llaves decoy públicas seleccionadas en la cadena de bloques para parecerse a las reales. Esto frusta a quienes buscan vigilar y rastrear transacciones en Monero. Todo esto sucede al mismo tiempo que el software de Monero se ejecuta de manera rápida y sin interrupciones, dando una velocidad óptima y un tamaño de cadena de bloques razonable.
+
+### _Desafíos_
+
+Ocultar al remitente en Monero no es perfecto. Diez decoys no siempre son suficientes, ya que algún TXO aleatorio puede ser identificable. Puede que con conocimiento público, los TXO, hayan sido utilizados en algún otro gasto. La existencia de las listas negras o blacklists muestra que esto puede ser un problema potencial y le da una solución a aquellos que esten motivados a esquivar y a agregar esta lista en sus monederos. Existen otras maneras de identificar los decoys en una transacción, como por ejemplo, atar los TXO de los pools y los TXO de las casas de cambio, estimando su validez en otras transacciones y utilizando tiempos de probabilidad.
+
+Imagina un alineamiento policial con un acusado injustamente y 10 decoys. Si los decoys fueran conocidos a un testigo falso, no serviría de nada proteger al acusado. Esto también se confirma con las firmas de anillo de Monero; ocultarse entre decoys solo sirve cuando los decoys son extraños y, para algunos adversarios, en particular adversarios con recursos, solo 10 decoys aleatorios pueden ser insuficiente. Y entonces, ¿por qué no usar más?
+
+Bueno, para el algoritmo MLSAG/CLSAG utilizado por Monero, tanto el tamaño de la transacción como el tiempo para procesarla crecen linealmente con la cantidad de decoys (según su complejidad algorítmica Big O). La utilización de diez decoys compensa la necesidad de privacidad con la necesidad de una cadena de bloques pequeña y un procesamiento más rápido. Incrementar el número de decoys puede ayudar a mejorar la privacidad pero también incrementa el tamano de la transacción, que de por si ya es un poco más grande que las transacciones de Bitcoin, y ralentizar la validación de los monederos.
```

Comment:
Other grammar adjustments and a correction:

_Incrementar el número de decoys puede ayudar a mejorar la **privacidad,** pero también incrementa el **tamaño** de la transacción, que de por **sí** ya es un poco más grande _

Reply:
Fixed :.)